chips/sam4l/src/crccu.rs

```diff
@@ -407,4 +399,4 @@ impl<'a> crc::CRC for Crccu<'a> {
 }
 
 /// Static state to manage the CRCCU
-pub static mut CRCCU: Crccu<'static> = Crccu::new();
+pub static mut CRCCU: Crccu<'static> = Crccu::new(BASE_ADDRESS);
```

Comment:
I don't understand why is better to have `BASE_ADDRESS` as argument here when it's just a `fixed value`? SAM4L only has one `CRCCU` right?!

Reply:
I don't know? I just copied the first thing I found in other SAM4L drivers.